### PR TITLE
music-manager.rb: do not move prefpane

### PR DESCRIPTION
**Pull request by vitorgalvao**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Why would we be taking it? Presumably the app will take care of it.

### CODE REVIEW

**Comment by tjanson** [comment]
See #20405:

> The app will copy the pref pane itself on first run (if it does not already exist), but then brew uninstall will not remove it.
> By letting brew symlink the pref pane, it uninstalls properly.

**Comment by tsparber** [comment]
@tjanson Would a `uninstall delete: '~/Library/Preference Panes/MusicManager.prefPane'` also work in this case?

**Comment by vitorgalvao** [comment]
@tjanson I was going to suggest what @tsparber just mentioned. As to why I’m removing this, see https://github.com/caskroom/homebrew-cask/issues/21657#issuecomment-223638491. Since we now move instead of linking, this current cask essentially modifies the `app` (since it removes the `prefpane` from inside) which isn’t good at all.

File: Casks/music-manager.rb [hunk 1]
```diff
@@ -10,4 +10,6 @@
   # Renamed for consistency: app name is different in the Finder and in a shell.
   # Original discussion: https://github.com/caskroom/homebrew-cask/pull/4282
   app 'MusicManager.app', target: 'Music Manager.app'
+
+  uninstall delete: '~/Library/Preference Panes/MusicManager.prefPane'
```
Comment (tjanson) [inline Casks/music-manager.rb]: This should be `~/Library/PreferencePanes/…` (no space), right?

**Comment by tjanson** [comment]
Ah, cool to see that the move to move has been made. :+1:
Removing the pane during uninstall works for me, but the path is `"~/Library/PreferencePanes/MusicManager.prefPane"`.

File: Casks/music-manager.rb (hunk 1 above)
Comment (tsparber) [inline Casks/music-manager.rb]: I just used the path from `man brew-cask` without checking. Turns out this needed some fixing (#21670).

**Pull request merged**